.env.j2

```diff
@@ -0,0 +1,17 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Required to access Trello's API: https://trello.com/app-key 
+TRELLO_KEY={{ trello_key }}
+TRELLO_TOKEN={{ trello_token }}
+
+# Required to access the TRELLO BOARD
+TRELLO_BOARD_ID=64f301eaccf7f0177989febc
```

Comment:
Not super important, but worth noting that if we had multiple environments we'd probably want this to be a variable too - since it's not sensitive we could always keep this value set as a variable default on the playbook to save us having to constantly copy it in